src/script/user/UserRepository.ts

```diff
@@ -586,7 +574,27 @@ export class UserRepository extends TypedEventEmitter<Events> {
     );
   }
 
-  private mapUserResponse(found: APIClientUser[], failed: QualifiedId[], dbUsers?: UserRecord[]): User[] {
+  // Replaces a deleted user name ("default") with the name from the local database.
+  private replaceDeletedUserName(apiUsers: APIClientUser[], dbUsers: UserRecord[]): UserRecord[] {
```

Comment:
Just a minor suggestion, completely your call.
```suggestion
  private replaceDeletedUserNames(apiUsers: APIClientUser[], dbUsers: UserRecord[]): UserRecord[] {
```

Reply:
Yeah, it's gonna be better, thanks :) 

Also, I've changed `replace` to `restore` - it should be more self-descriptive now